Raw json:

{'hasError': True, 'errParameter': ['alluxio.user.network.max.inbound.message.size', 'alluxio.worker.network.writer.buffer.size.messages'], 'reason': ['The value of alluxio.user.network.max.inbound.message.size is not a valid number.', 'The value of alluxio.worker.network.writer.buffer.size.messages is not a valid number.']}

{'hasError': True, 'errParameter': ['alluxio.user.network.max.inbound.message.size', 'alluxio.worker.network.writer.buffer.size.messages'], 'reason': ['The value of alluxio.user.network.max.inbound.message.size is not a valid number.', 'The value of alluxio.worker.network.writer.buffer.size.messages is not a valid number.']}

{'hasError': True, 'errParameter': ['alluxio.user.network.max.inbound.message.size'], 'reason': ['alluxio.user.network.max.inbound.message.size should be a positive integer.']}

Final result:

There are 2 misconfiguration parameters in the input: alluxio.user.network.max.inbound.message.size	alluxio.worker.network.writer.buffer.size.messages
[Ciri] Reason for alluxio.user.network.max.inbound.message.size: The value of alluxio.user.network.max.inbound.message.size is not a valid number.
[Ciri] Reason for alluxio.worker.network.writer.buffer.size.messages: The value of alluxio.worker.network.writer.buffer.size.messages is not a valid number.
